apps/dialtone-documentation/docs/components/scrollbar.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Scrollbar
+description: A directive that adds customized overlay scrollbars to any scrollable section.
+status: beta
+thumb: true
+image: assets/images/components/scroller.png
+---
+
+## Scrollbar directive
+
+Allows to add overlay scrollbars that will look the same for every browser. The directive sets up the scrollbars from the library [OverlayScrollbars](https://kingsora.github.io/OverlayScrollbars/).
+
+<code-well-header>
+  <div class="d-hmx164 d-w30p d-bar8 d-ba d-bc-black-300" v-dt-scrollbar>
+    <div v-for="item in items" class="item">
+      {{ item}}
+    </div>
+  </div>
+</code-well-header>
+
+<code-example-tabs
+htmlCode='
```

Comment:
Since this is purely a Vue plugin, I wonder how much value there is in showing HTML code beyond seeing a sample rendered result. No one will be copying the HTML code. 

Just a thought.

Reply:
I think in this case is valuable precisely to see the result, yes.